app/Http/Controllers/UploadController.php

```diff
@@ -30,7 +30,7 @@ public function upload(Request $request)
                 $file = File::save(
                     $uploaded,
                     $request->client->name,
-                    (isset($request->private) && $request->private !== 'false'),
+                    ($request->has('private') && $request->private !== 'false'),
```

Comment:
$request->boolean('private', false)

Reply:
This is done so that you only need to add `?private` to the query. At the same time, with `?private=false` the file must be public.

I have tested several ways and have not found a better solution.